src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -397,7 +397,7 @@ public String getZookeeperPath() {
     }
 
     public String getGsCredentialsPath() {
-        return getString("secor.gs.credentials.path");
```

Comment:
We usually prefer people providing default value through properties file, we want to make the default value more explicit.  Can you provide the default value in secor.common.properties?